extension/content.css

```diff
@@ -196,7 +196,7 @@
 }
 
 /* indicate collapsible file headers with zoom cursor */
-.file-header[data-path], .file-header[data-path] .diffstat {
+.js-details-container .file-header[data-path], .file-header[data-path] .diffstat {
```

Comment:
@DrewML This still does the same for me in the inline diff in #112.

However this works for me:

``` css
.js-details-container:not(.discussion-item) .file-header[data-path], .file-header[data-path] .diffstat {
	cursor: zoom-out !important;
}
.refined-github-minimized:not(.discussion-item) .file-header[data-path], .refined-github-minimized .file-header[data-path] .diffstat {
	cursor: zoom-in !important;
}
```